context.trakt.watched/addon.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="context.trakt.watched" name="Trakt - Watched button" version="1.0.3" provider-name="Razzeee">
+  <requires>
+    <import addon="xbmc.python" version="2.20.0"/>
+    <import addon="script.trakt" version="3.0.2"/>
+  </requires>
+    <extension point="kodi.context.item" library="addon.py">		
+      <item>
+        <label>32000</label>
+        <visible>String.IsEqual(ListItem.dbtype,movie) | String.IsEqual(ListItem.dbtype,tvshow) | String.IsEqual(ListItem.dbtype,season) | String.IsEqual(ListItem.dbtype,episode)</visible>
+      </item>
+    </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="de">Trakt-Schaltfläche für Gesehene zum Kontextmenü in der Bibliothek hinzufügen.</summary>
+    <summary lang="en">Add Trakt watched button to context menu in library</summary>
```

Comment:
Haven't noticed at first. Default language code is `en_GB`

Reply:
same for de_DE :)